Jenkinsfile

```diff
@@ -57,7 +57,9 @@ pipeline {
                         docker.build("javaduck:${env.BUILD_ID}", "aggregation-worker/")
                         withEnv(['PATH+EXTRA=/var/lib/jenkins/bin']) {
                             docker.withRegistry(env.ECR_REGISTRY_URL) {
-                                docker.image("javaduck:${env.BUILD_ID}").push("latest")
+                                dockerImage = docker.image("javaduck:${env.BUILD_ID}")
```

Comment:
Should probably just use BUILD_NUMBER here for consistency: https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#using-environment-variables

>BUILD_ID
    The current build ID, identical to BUILD_NUMBER for builds created in Jenkins versions 1.597+

The doco here: https://www.jenkins.io/doc/book/pipeline/docker/ has an example where push is run with no parameters. I'm wondering if we should just be doing something simpler, e.g.

```groovy
dockerImage = docker.image("javaduck:${env.BUILD_ID}")
dockerImage.push()
dockerImage.push("latest")
```

It seems we're building with one local name, pushing to a different remote name and then tagging latest.

Reply:
Or if it doesn't work as expected, or gets overcomplicated, we can just run with it, but changingBUILD_ID to BUILD_NUMBER at least for clarity that they are actually the same thing.